### PR TITLE
Radd fifth delivery

### PR DESCRIPTION
**Pull request by francosiNTTDATA**
This is the last delivery to complete all expected flows.

### CODE REVIEW

File: src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java [hunk 1]
```diff
@@ -174,4 +174,25 @@ public void waitErrorMessageIdOperazione(String errorText) {
     public void insertOperationID(String idOperazioneErroreAAR) {this.operationIdDAATextField.sendKeys(idOperazioneErroreAAR);}
     public void selectAvvisiAvvenutaRicezioneButton() { this.AARRadioButton.click();}
     public void clickPersonaGiuridicaRadioButton() {this.personaGiuridicaRadioButton.click();}
+
+    public int getNRighe2(String partitaIVAPGAA) {
+        try {
+            By rigaBy = By.xpath("//tr[td[contains(text(),'MNZVMH95B09L084U')]]");
```
Comment (leleOFA) [inline src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java:180]: using xpath to take id that contains text

File: src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java [hunk 2]
```diff
@@ -174,4 +174,25 @@ public void waitErrorMessageIdOperazione(String errorText) {
...
+            this.getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(rigaBy));
+            return this.elements(rigaBy).size();
+        }catch (TimeoutException e){
+            return 0;
+        }
+    }
+
+    public void clickRichiestaRestituita1(String partitaIVAPGAA) {
+        try {
+            By richiestaBy = By.xpath("//tr[td[contains(text(),'MNZVMH95B09L084U')]]");
```
Comment (leleOFA) [inline src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java:190]: as before

File: src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java (hunk 1 above)
Comment (francosiNTTDATA) [reply src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java:180]: there is no ID for td tag: class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-155o2nr"

File: src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java (hunk 2 above)
Comment (francosiNTTDATA) [reply src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java:190]: Same

File: src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java (hunk 1 above)
Comment (leleOFA) [reply src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java:180]: let's put it in the list of components to add the ids to

**Review by leleOFA — APPROVED** [review]

**Pull request merged**